src/mpid/ch4/netmod/ofi/ofi_init.h

```diff
@@ -993,10 +993,9 @@ static inline int MPIDI_NM_get_local_upids(MPIR_Comm * comm, size_t ** local_upi
     goto fn_exit;
 }
 
-static inline int MPIDI_OFI_upids_to_lupids_general(int size,
-                                                    size_t * remote_upid_size,
-                                                    char *remote_upids,
-                                                    int **remote_lupids, int use_av_table)
+static inline int MPIDI_NM_upids_to_lupids(int size,
+                                           size_t * remote_upid_size,
+                                           char *remote_upids, int **remote_lupids)
```

Comment:
Actually, this change is related to the current commit. The original `MPIDI_OFI_upids_to_lupids_general` use different code paths for `FI_AV_MAP` and `FI_AV_TABLE`. Therefore it needs the `MPIDI_NM_upids_to_lupids` as a wrapper for the ADI.

As a result of this commit, both modes can use the exactly same code path. Therefore, the wrapper should go away. So I figured the renaming should be in this commit as well.